graviti/dataframe/frame.py

```diff
@@ -140,6 +163,38 @@ def _construct(
         obj._index = index
         return obj
 
+    @classmethod
+    def _load_repr_info(
```

Comment:
Can we make this method constant? (A method will not change the input argument?)